content/references/xrpljs2-migration-guide.jp.md

```diff
@@ -0,0 +1,344 @@
+
+どちらのメソッドに関しても、準備済みトランザクション説明と[`Wallet`インスタンス](#keys-and-wallets)をパスすることによって、署名済みトランザクションをメソッドに直接パス、もしくは、送信直前にトランザクションに署名することができます。
+
+```js
+const tx_json = await client.autofill({
+  "TransactionType": "AccountSet",
+  "Account": wallet.address, // "wallet"はWalletクラスのインスタンス
+  "SetFlag": xrpl.AccountSetAsfFlags.asfRequireDest
+})
+try {
+  const submit_result = await client.submitAndWait(tx_json, wallet)
+  // submitAndWait() はトランザクションの結果が確定するまでreturnしません。
+  // トランザクションがネットワークに確認されなかった場合、XrplErrorが発生します。
+  // ディザスタリカバリには対応しません。
+  console.log("Transaction result:", submit_result)
+} catch(err) {
+  console.log("Error submitting transaction:", err)
+}
+```
+
+Alternatively, you can use the `sign` method of a wallet to sign a transaction and then use `submitAndWait(tx_blob)` to submit it. This can be useful for building [reliable transaction submission](reliable-transaction-submission.html) that can recover from power outages and other disasters. (The library does not handle disaster recovery on its own.)
+もしくは、トランザクション署名のためにwalletの`sign`メソッドを、送信のために`submitAndWait(tx_blob)`を使用することができます。
+停電やその他災害から復旧させる[信頼できるトランザクションの送信](reliable-transaction-submission.html)のビルドに便利です。(ライブラリは単独でディザスタリカバリに対処しません。)
+
+### LastLedgerSequenceのコントロール
+
+In ripple-lib 1.x, you could specify a `instructions.maxLedgerVersionOffset` when preparing a transaction to define the `LastLedgerSequence` parameter of the prepared transaction as being some number of ledgers _after_ the latest validated one at the time. In 2.0, you can do this by looking up the latest validated ledger index, then specifying the `LastLedgerSequence` explicitly before auto-filling the transaction.
+ripple-lib 1.xでは、トランザクションを準備し、準備済みトランザクションの`LastLedgerSequence`パラメータを、その時点で最新の検証済みレジャー番号 _以降_ のレジャー番号を指定する際、`instructions.maxLedgerVersionOffset`を利用できました。2.0では、最新の検証済みレジャー番号を調べ、トランザクションのオートフィル前に`LastLedgerSequence`を明確に指定することが可能です。
+
+**xrpl.js 2.0:**
+
+```js
+const vli = await client.getLedgerIndex()
+
+const prepared = await client.autofill({
+  "TransactionType": "Payment",
+  "Account": sender,
+  "Amount": xrpl.xrpToDrops("50.2"),
+  "Destination": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+  "LastLedgerSequence": vli+75 // デフォルトの1分以内ではなく5分以内
+})
+```
+
+旧メソッド同様、デフォルトでは、`Client.autofill()`は合理的な`LastLedgerSequence`値を提示します。`LastLedgerSequence`フィールドが _ない_ トランザクションを準備するには、`null`値の`LastLedgerSequence`を提示します:
+
+```js
+const prepared = await client.autofill({
+  "TransactionType": "Payment",
+  "Account": sender,
+  "Amount": xrpl.xrpToDrops("50.2"),
+  "Destination": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+  "LastLedgerSequence": null // トランザクションは有効期限切れになりません
+})
+```
+
+
+## キーおよびウォレット
+
+xrpl.js 2.0 introduces a new [`Wallet` class](https://js.xrpl.org/classes/Wallet.html) for managing [cryptographic keys](cryptographic-keys.html) and signing transactions. This replaces functions that took seed or secret values in ripple-lib 1.x, and handles various address encoding and generation tasks as well.
```

Comment:
I think this sentence that is also translated below so the English version should be removed.